calender/calendar_tui.py

```
@@ -1264,113 +1268,137 @@ def draw_attendee_details(self):
             # Top border
             self.stdscr.addstr(start_y, start_x, "╔" + "═" * (modal_width - 2) + "╗", curses.color_pair(1) | curses.A_BOLD)
 
-            # Title
-            title = f" Attendees for {event.summary[:40]} "
+            # Title bar
+            title = f" Event Details "
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)